setup.cfg

```diff
@@ -22,8 +22,15 @@ install_requires =
     pygments>=2.8.1
```

Comment:
Decided to list each development and linting requirement in this file on separate lines to improve branch diffs and provider better git blames in the future